_posts/2020-01-06-2019-reading-in-review.md

```diff
@@ -0,0 +1,501 @@
+written in. It's written in what Hoban imagined English might become if society
+fell and education evaporated and such. It's hard to understand at first, but
+you get it pretty quickly. However, now that I know a fair amount about language
+change… this was really bothersome to me because the language change is more
+like if someone were very drunk than how language actually changes. Ah well.
+
+
+### _The Long Way to a Small, Angry Planet_ by Becky Chambers (???)
+
+The best way I can talk about this book is to say that it is everything that
+fans of _Firefly_ think _Firefly_ is (where it is actually not very those
+things). It is about found families and acceptance and healing and it takes
+place on a spaceship that is also a home. It's so lovely.
+
+
+### _Jacobin 31: Breaking Bank_ (???)
+
+This is a quarterly socialist magazine. I have a note that this volume was about
+economics. Skimming through the table of contents just now, none of the pieces
+jump out at me as being amazing, but I do recall that it helped me understand
+what "the neoliberalization of economies" means and how it negatively impacts
+working people.
+
+
+### _Apocalypse Nyx_ by Kameron Hurley (2019-02-06)
+
+I've read all the novels in the _Bel Dame Apocrypha_ and liked them. They're all
+a buggy, gross, magical, self-destructive scifi mess in the best way. This book
+is a collection of short stories in that continuity that doesn't dissapoint. I
+think they were largely published only digitally originally, which is why I'd
+missed them. I tend to prefer dead tree reading, so it was nice for them to be
+collected such.
+
+
+### _Washington Black_ by Esi Edugyan (2019-02-19)
+
+My wife bought this for me after hearing a review of it on, I think, NPR. My
+impression is that she thought it would be more science fiction than it was.  It
+ended up not being much at all, but I still enjoyed it all the way up until the
+end. I just incredibly did not understand what the author was trying to do or
+say with the ending.
+
+
+### _The ABCs of Capitalism_ by Vivek Chibber (2019-02-21)
+
+This is actually three pamphlets published by Jacobin: _A: Understanding
+Capitalism_, _B: Capitalism & the State_ and _C: Capitalism & Class Struggle_.
+I've since loaned these out to a few people and I wish more folks would read
+them. The _B_ volume in particular does a great job drawing a connection from
+capitalism on a fundemental structural level to injustice and explaining why the
+one increasingly causes the other over time. I think they also do a good job of
+making the point that there is only so far we can get to achieve justice for all
+without overthrowing capitalism and, thus, the limits we're generally under if
+we constrain our efforts to electoral politics.
+
+
+### _Star Wars: Last Shot_ by Daniel José Older (2019-03-04)
+
+I think this is a story about Lando and Han? There's a non-binary character in
+it who was pretty cool, I think? And maybe it hopped between eras of, like,
```

Comment:
Do you think it's pretty cool that there's a non-binary character or do you think the character is non-binary?